src/Propel/Generator/Platform/Util/MysqlUuidMigrationBuilder.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * MIT License. This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Propel\Generator\Platform\Util;
+
+use Propel\Generator\Model\Column;
+use Propel\Generator\Model\Index;
+use Propel\Generator\Platform\MysqlPlatform;
+
+/**
+ * Creates migration statements for UUID columns in MySQL.
+ *
+ * @phpstan-consistent-constructor
+ */
+class MysqlUuidMigrationBuilder
```

Comment:
Yes, I'd like that too.  But since the builtin function to convert UUIDs is only available in MySQL, other platforms, cannot use a pure SQL migration. I think we would have do something like creating the new column, read all data into PHP and then fill the new column with an INSERT, before we can replace the old column. Or write function declarations for the other platforms. Both is a bit too much for now. Maybe someday...